package/MDAnalysis/analysis/density.py

```diff
@@ -384,6 +390,16 @@ def __init__(self, atomgroup, delta=1.0,
         self._zdim = zdim
 
     def _prepare(self):
+        """Function to prepare Density
+
+        Raises
+        ------
+        ValueError
+            if AtomGroup is empty and no user defined grid is provided
+        UserWarning
+            if AtomGroup is empty and a user defined grid is provided
+
+        """
```

Comment:
I guess I forgot to add a 'r' symbol which might be what triggers building the docs. My next commit will include this! :)

Reply:
Hi @orbeckst, as I am just starting out with this kind of development. I am not sure how the webpage docs are built with the repository code? Perhaps there is a package I should refer to see how it's built so that I can ensure this comment is included in the webpage as well?

Reply:
@aditya-kamath you can check updates to the PR in the [ReadTheDocs](https://mdanalysis--3091.org.readthedocs.build/en/3091/documentation_pages/analysis/density.html#MDAnalysis.analysis.density.DensityAnalysis) build. Otherwise, you could ``cd package/doc/sphinx``, type ``make html``, and your output will be in ``../html``.

Reply:
Thanks! I think this issue is related to sphinx docs. Hopefully my next commit will compile this to the docs build.